BillManagementSystem/accounting/schema/types.py

```diff
@@ -0,0 +1,106 @@
+import strawberry
+from strawberry import auto, ID
+import strawberry_django
+from typing import List, Union
+from datetime import date
+from .. import models
+from django.contrib.auth.models import User as DJangoUser
+import decimal
+from strawberry_django_plus import gql
+
+
+@strawberry.django.type(models.IndividualSpending)
+class IndividualSpending:
+    id: ID
+    user: "User"
+    amount: decimal.Decimal
+    note: auto
+    title: auto
+    bill: "Bill"
+
+
+@strawberry.django.type(models.Bill)
+class Bill:
```

Comment:
You can rename these to be `BillScalar`, `UserScalar` etc. to reduce confusion in code